ADOPTERS.md

```diff
@@ -58,6 +58,9 @@ Okteto integrates Velero in [Okteto Cloud][94] and [Okteto Enterprise][95] to pe
 **[Replicated][100]**<br>
 Replicated uses the Velero open source project to enable snapshots in [KOTS][101] to backup Kubernetes manifests & persistent volumes. In addition to the default functionality that Velero provides, [KOTS][101] provides a detailed interface in the [Admin Console][102] that can be used to manage the storage destination and schedule, and to perform and monitor the backup and restore process.
 ​
+**[CloudCasa][103]**  
```

Comment:
![image](https://user-images.githubusercontent.com/11228024/151410400-93c4780a-c895-4cf7-af10-d5d8b96893a7.png)

Might want to check the resulting formatting. Shouldn't there be a new line before org name?